sov-modules/sov-modules-api/src/dispatch.rs

```diff
@@ -23,3 +23,14 @@ pub trait DispatchCall {
         context: &Self::Context,
     ) -> Result<CallResponse, Error>;
 }
+
+/// A trait that needs to be implemented for any query message.
+pub trait DispatchQuery {
+    type Context: Context;
+
+    /// Dispatches a query message to the appropriate module.
+    fn dispatch(
```

Comment:
Should we name this `dispatch_query` instead of `dispatch` so that we don't need to qualify it with the trait name everywhere?

Reply:
Done!